### PR TITLE
Model tensorflow

### PR DESCRIPTION
**Pull request by benjaminmielke**
Added a method that builds an ANN with tensorflow and uses Pinball Loss function to created quantile regression predictions on the demand and PV. 

Adjusted the evaluate_model() method to be re-usable

### CODE REVIEW

**Review by rjbergerud — APPROVED** [review]
At some point when we have multiple models going, we might want to have a dictionary attached to the PodChallenge object that stores several different trained classifiers (according to model and parameters)
